### PR TITLE
Remove cloud component packages

### PR DESCRIPTION
**Pull request by ZitaNemeckova**

### CODE REVIEW

**Comment by ZitaNemeckova** [comment]
@himdel

File: src/Utilities.scss [hunk 1]
```diff
@@ -1,3 +0,0 @@
-// Sass Imports
-
-@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all';
```
Comment (himdel) [inline src/Utilities.scss:3]: This was setting 

```
button:focus {
    outline:none;
}
```

should we keep that?

File: src/components/loading/loading-with-header.tsx [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
-          <PageHeaderTitle
-            title={<Skeleton size='sm'></Skeleton>}
-          ></PageHeaderTitle>
-        </PageHeader>
+        <section
+          className={
+            'pf-l-page-header pf-c-page-header pf-l-page__main-section pf-c-page__main-section pf-m-light'
+          }
+        >
+          <Title headingLevel='h1'>
```
Comment (himdel) [inline src/components/loading/loading-with-header.tsx:16]: missing `size="2x1"`?

File: src/components/loading/loading-with-header.tsx [hunk 3]
```diff
@@ -1,29 +1,26 @@
...
+            <Skeleton></Skeleton>
```
Comment (himdel) [inline src/components/loading/loading-with-header.tsx:17]: Note to self: test (different implementation)

**Comment by himdel** [comment]
Code looks LGTM :+1: 

These seem to change implementation, need to check visually: 
* [ ] `Skeleton`
* [ ] `Spinner` (:+1 for having one less spinner version :))
* [ ] `Section`

File: src/components/headers/base-header.tsx [hunk 4]
```diff
@@ -51,7 +50,7 @@ export class BaseHeader extends React.Component<IProps, {}> {
               />
             ) : null}
             <div>
-              <PageHeaderTitle title={title} />
+              <Title headingLevel='h1' size='2xl' children={title} />
```
Comment (himdel) [inline src/components/headers/base-header.tsx:53]: `size='2x1'` is not technically needed, it defaults to that for h1

**Review by himdel — APPROVED** [review]
Works for me :+1:  (just needs a rebase)

**Pull request merged**